### PR TITLE
Update from changes for Canvas GA

### PR DESCRIPTION
**Pull request by atangOSIsoft**
These changes are for updates in the Assets feature for Canvas GA.

### CODE REVIEW

File: Content_Portal/Documentation/Assets/asset-centric-data-api.md [hunk 1]
```diff
@@ -53,6 +53,9 @@ The namespace identifier
 `string assetId`  
 The asset identifier
 
+[optional] `string[] streams`   
+An optional parameter representing a comma separated list of streams which you are interested in. By default, all data calls return data for all stream references.
```
Comment (lasato-OSI) [inline Content_Portal/Documentation/Assets/asset-centric-data-api.md:57]: If I understand this correctly, the parameter is a comma-separated list of streams. If that's true, then could we say:

An optional parameter consisting of a comma-separated list of streams that you are interested in.

File: Content_Portal/Documentation/Assets/asset-resolved-asset.md [hunk 2]
```diff
@@ -147,3 +147,43 @@ Content-Type: application/json
...
+
+`string namespaceId`  
+The namespace identifier
+
+#### Request body 
+
+A list of asset `Id`s.
+
+#### Example request body 
+Requesting resolved assets for assets with id, "Asset1", "Asset2", and "Asset3".
```
Comment (lasato-OSI) [inline Content_Portal/Documentation/Assets/asset-resolved-asset.md:171]: Requesting resolved assets for assets with the following IDs: "Asset1", "Asset2", and "Asset3".

File: Content_Portal/Documentation/Assets/asset-search-api.md [hunk 3]
```diff
@@ -156,13 +156,13 @@ Content-Type: application/json
...
+| Metadata/Name:Building*      | Returns all assets with at least one metadata name whose description contains the string **Building**. |
+| Metadata/Description:heater* | Returns all assets with at least one metadata whose description starts with **heater**. |
+| Metadata/Value:123           | Returns all assets with at least one metadata whose Value property equals **123**. |
 | Id:X* AND Metadata/Name:B*   | Returns all assets with `Id` starting with **X** and containing at least one metadata value with a name that starts with a **B**. |
 | AssetTypeId:HeaterTypeId | Returns all assets with `AssetTypeId` matching `HeaterTypeId` |
-| AssetTypeName:HeaterTypeName | Returns all assets whose Name field of the asset type matches `HeaterTypeName` |
-
-
+| AssetTypeName:HeaterTypeName | Returns all assets whose `Name` field of the asset type matches **HeaterTypeName** |
+| StreamProperties:Pressure | Returns all assets whose stream reference has a stream property id of **Pressure**. Note that search for Sds stream property id will only search for non-key stream properties. |
```
Comment (lasato-OSI) [inline Content_Portal/Documentation/Assets/asset-search-api.md:167]: Returns all assets whose stream reference with the stream property Id **Pressure**. Note: Sds stream property Ids searches only search non-key stream properties.

File: Content_Portal/Documentation/Assets/asset-search-api.md [hunk 4]
```diff
@@ -156,13 +156,13 @@ Content-Type: application/json
...
+| StreamReferences:Name1| Returns all assets whose stream references contain a stream reference name matching **Name1**. |
```
Comment (lasato-OSI) [inline Content_Portal/Documentation/Assets/asset-search-api.md:168]: Returns all assets whose stream references contain a stream reference that matches **Name1**. 

(This one is a nit.)

File: Content_Portal/Documentation/Assets/asset-status.md [hunk 5]
```diff
@@ -82,7 +81,9 @@ The asset or asset type's 'StreamReferences' field has an 'Id' property. To assi
 
 ## `Get Asset Status`
 
-View the status of an asset
+View the status of an asset. 
+
+The status of an asset is determined by exact match of the Sds stream property value to the value of the ValueStatusMapping. If there are no exact matches, the status is assigned a 0, or "Unknown".
```
Comment (lasato-OSI) [inline Content_Portal/Documentation/Assets/asset-status.md:86]: The status of an asset is determined by an exact match of the Sds stream property value to the value of the ValueStatusMapping. If there are no exact matches, the status is a 0 (Unknown).

File: Content_Portal/Documentation/Assets/assets.md [hunk 6]
```diff
@@ -15,14 +15,16 @@ In this situation, an asset type can be used to create multiple similar assets.
...
-| TypeReferences | Type Reference List | Optional  | No*        | Asset type type references                                     | No | Yes            |
-| StatusMapping | Status Mapping | Optional  | No*        | Asset and asset type status mapping | Yes | Yes            |
+| Metadata      | Metadata List     | Optional  | Yes       | Asset and asset type metadata                               | Yes  | Yes            |
+| StreamReferences   | Stream Reference List | Optional  | No       | Asset stream references                                             | Yes  | No            |
+| TypeReferences | Type Reference List | Optional  | No        | Asset type type references                                     | No | Yes            |
+| StatusMapping | Status Mapping | Optional  | No        | Asset and asset type status mapping | Yes | Yes            |
+
+* `Id` is not required on property if the `Name` matches a `Name` on the Asset Type property. In this case, the `Id` of the property on the Asset will be set as the property `Id` of the Asset Type. This also applies when an Asset is updated.
```
Comment (lasato-OSI) [inline Content_Portal/Documentation/Assets/assets.md:27]: `Id` is not required on property if the `Name` matches a `Name` on the asset type property. In this case, the `Id` of the property on the asset is set as the property `Id` of the asset type. This also applies when an asset is updated.

File: Content_Portal/Documentation/Assets/asset-search-api.md (hunk 3 above)
Comment (atangOSIsoft) [reply Content_Portal/Documentation/Assets/asset-search-api.md:167]: After reading this, it sounds a bit funny, perhaps:

Returns all assets which has one or more stream references with the Sds stream property Id Pressure. Note: This search only searches non-key Sds stream properties.

File: Content_Portal/Documentation/Assets/asset-centric-data-api.md (hunk 1 above)
Comment (atangOSIsoft) [reply Content_Portal/Documentation/Assets/asset-centric-data-api.md:57]: Apologize that this wasn't clear. This is a comma separated list of stream reference names

File: Content_Portal/Documentation/Assets/asset-resolved-asset.md (hunk 2 above)
Comment (atangOSIsoft) [reply Content_Portal/Documentation/Assets/asset-resolved-asset.md:171]: Sounds good. Thanks!

File: Content_Portal/Documentation/Assets/asset-status.md (hunk 5 above)
Comment (atangOSIsoft) [reply Content_Portal/Documentation/Assets/asset-status.md:86]: Thanks, that sounds better to me.

File: Content_Portal/Documentation/Assets/assets.md (hunk 6 above)
Comment (atangOSIsoft) [reply Content_Portal/Documentation/Assets/assets.md:27]: Thanks!

**Review by atangOSIsoft — COMMENTED** [review]
Thanks for the corrections

File: Content_Portal/Documentation/Assets/asset-search-api.md (hunk 4 above)
Comment (atangOSIsoft) [reply Content_Portal/Documentation/Assets/asset-search-api.md]: A stream reference in an Asset contains a few properties, of them, name and id.  I was thinking that adding stream reference **name** was more clear, but ok with the update if it's already clear.

File: Content_Portal/Documentation/Assets/assets.md (hunk 6 above)
Comment (atangOSIsoft) [reply Content_Portal/Documentation/Assets/assets.md:27]: Should be on the metadata and stream references object, not the assets section

File: Content_Portal/Documentation/Assets/asset-centric-data-api.md (hunk 1 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/asset-centric-data-api.md:57]: Done. I also updated descriptions of this parameter throughout.

File: Content_Portal/Documentation/Assets/asset-resolved-asset.md (hunk 2 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/asset-resolved-asset.md:171]: Done

File: Content_Portal/Documentation/Assets/asset-search-api.md (hunk 3 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/asset-search-api.md:167]: Done

File: Content_Portal/Documentation/Assets/asset-status.md (hunk 5 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/asset-status.md:86]: Done

File: Content_Portal/Documentation/Assets/asset-search-api.md (hunk 4 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/asset-search-api.md]: Done

File: Content_Portal/Documentation/Assets/assets.md (hunk 6 above)
Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/assets.md]: No edits required. Done

**Pull request merged**